src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -0,0 +1,161 @@
+                getItem.setId(resultSet.getLong("item_id"));
+                return Optional.of(getItem);
+            }
+        } catch (SQLException e) {
+            logger.warn("Can't find item by id - " + id);
+        } finally {
+            if (statement != null) {
+                try {
+                    statement.close();
+                } catch (SQLException e) {
+                    logger.warn("Can't close statement!");
+                }
+            }
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public void update(Item entity) {
+        String query = String.format(Locale.ROOT,
+                "UPDATE %s SET name = %s, price = %.4f WHERE item_id = %d;",
+                DB_TABLE_NAME, entity.getName(),entity.getPrice(), entity.getId());
+        try {
+            statement = connection.createStatement();
+            if (statement.executeUpdate(query) == 1) {
+                logger.info("Success update");
+            }
+        } catch (SQLException e) {
+            logger.warn("Can't find item by id - " + entity.getId());
+        } finally {
+            if (statement != null) {
+                try {
+                    statement.close();
+                } catch (SQLException e) {
+                    logger.warn("Can't close statement!");
+                }
+            }
+        }
+    }
+
+    @Override
+    public boolean deleteById(Long id) {
+        String query = String.format("DELETE FROM %s WHERE item_id = %d;", DB_TABLE_NAME, id);
+        try {
+            statement = connection.createStatement();
+            if (statement.executeUpdate(query) == 1) {
+                logger.info("Success delete");
+                return true;
+            }
+        } catch (SQLException e) {
+            logger.warn("Can't find item by id - " + id);
+        } finally {
+            if (statement != null) {
+                try {
+                    statement.close();
+                } catch (SQLException e) {
+                    logger.warn("Can't close statement!");
+                }
+            }
+        }
+        return false;
+    }
+
+    @Override
+    public boolean delete(Item entity) {
+        return false;
+    }
+
+    @Override
+    public List<Item> getAll() {
+        String query = String.format("SELECT * FROM %s;", DB_TABLE_NAME);
+        List<Item> itemList = new ArrayList<>();
+        try {
+            statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery(query);
+            while (resultSet.next()) {
+
+                resultSet.getString("name");
+                resultSet.getDouble("price");
+
+                Item getItem = new Item(resultSet.getString("name"),
+                        resultSet.getDouble("price"));
+                getItem.setId(resultSet.getLong("item_id"));
+                itemList.add(getItem);
+            }
+        } catch (SQLException e) {
+            logger.warn("SQL error ");
```

Comment:
```suggestion
            logger.warn("Can't do something", e);
```
Логер має повертати якусь інформацію, що стосується дії, через яку стався Exception, та саму помилку. Або, ще краще завернути помилку в RuntimeException

Reply:
в інших логерах те ж саме